fronts-client/src/bundles/editionsIssueBundle.ts

```diff
@@ -17,3 +22,47 @@ export const {
 export type EditionsIssueState = State<EditionsIssue>;
 
 export default reducer;
+
+export const getEditionIssue = (
+  id: string
+): ThunkResult<Promise<void>> => async (dispatch: Dispatch) => {
+  try {
+    dispatch(actions.fetchStart());
+    const issuePromise = getIssueSummary(id);
+    const version = await getLastProofedIssueVersion(id);
+    const i = {
```

Comment:
Could this variable name be more descriptive?

Reply:
Done.